### ISSUE TITLE
Bug: Enabling more than 2 style flags does not work

### ISSUE THREAD

**Issue by Jofagi** [KeyWorksRW/wxUiEditor#1237]
### Description:
Creating a `wxTextCtrl` with style flags `wxTE_READONLY|wxTE_MULTILINE|wxTE_RICH2` is not possible, because code generation yields invalid code.

The generated code is  `wxTE_READONLY|wxTE_MULTILINE, wxTE_RICH2` (note the comma instead of pipe).

Example: [flags.wxui.txt](https://github.com/KeyWorksRW/wxUiEditor/files/12856427/flags.wxui.txt)

#### Observations: 

- The broken string is already in the wxui file, so it doesn't seem to be a bug in a specific code generator.
- It seems like only the first two flags are separated with a pipe. All additional flags get addd with commas as separators.
- `wxWindow`->`window_style` also has this problem. There are possibly more places...

**Comment by KeyWorksRW** [comment]
What version of wxUiEditor are you using? I ask because I tried this on version 1.1.2 (last release version) as well as the daily build and I could not reproduce the problem.

**Comment by Jofagi** [comment]
Yeah, it's not urgent for me (not that it would change anything :)).
I ended up using `wxStyledTextCtrl` and didn't run into that specific problem.

Anyway, thanks for this great tool. It saves me a lot of time and is pretty easy to use as well.
